client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -96,6 +97,33 @@ public void closeFlow(ClientConnection clientConnection) {
         flowIdReplyProcessorMap.remove(flow);
```

Comment:
When the client connection is closed , we close the Flow. In such a scenario we need to remove the corresponding writer from `idBatchSizeTrackerMap ` .

Reply:
done